pkg/config/config.go

```diff
@@ -145,6 +150,20 @@ func (cfg *Config) BindFlags() {
 		"Specific namespace the service controller will watch for object creation from CRD. "+
 			" By default it will listen to all namespaces",
 	)
+	flag.IntVar(
+		&cfg.DefaultDriftRemediationPeriod, flagDefaultDriftRemediationPerdiod,
```

Comment:
How about we call this flag `--reconcile-default-resync-seconds`?

Reply:
Done